systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -191,17 +187,17 @@ void testMirrorMaker2() {
 
         RollingUpdateUtils.waitTillComponentHasRolled(testStorage.getNamespaceName(), testStorage.getMM2Selector(), mirrorMakerReplicasCount, mm2PodsSnapshot);
 
-        // TODO: https://github.com/strimzi/strimzi-kafka-operator/issues/8864
+        // TODO https://github.com/strimzi/strimzi-kafka-operator/issues/8864
         // currently disabled for UTO, as KafkaTopic CR is not created -> we should check it directly in Kafka
-        if (!Environment.isKRaftModeEnabled() && !Environment.isUnidirectionalTopicOperatorEnabled()) {
+        /*if (!Environment.isKRaftModeEnabled()) {
```

Comment:
Any reason to comment this out?

Reply:
This would make the test fail. There are a number of test that still need to be fully converted to UTO as part of #8864.